mm2src/coins/utxo.rs

```diff
@@ -483,6 +485,51 @@ impl UtxoSyncStatusLoopHandle {
     }
 }
 
+#[derive(Debug, Default, Clone, Deserialize)]
+pub struct SPVConf {
+    // Determine if spv proof should be enable. Default to false.
+    pub enable_spv_proof: bool,
+    // Limit of block headers allowed to be stored in db..
+    max_stored_block_headers: Option<u64>,
+    // Starting block height and hex configuration.
+    start_block_header: Option<ElectrumBlockHeaderV14>,
+    /// The parameters that specify how the coin block headers should be verified. If None and enable_spv_proof is true,
+    /// headers will be saved in DB without verification, can be none if the coin's RPC server is trusted.
+    pub verification_params: Option<BlockHeaderVerificationParams>,
+}
+
+impl SPVConf {
+    pub fn max_stored_block_headers(&self) -> u64 { self.max_stored_block_headers.unwrap_or_default() }
+
+    pub fn starting_block_header_height(&self) -> u64 { self.start_block_header.clone().unwrap_or_default().height }
+
+    pub fn validate_verification_params(&self, coin: &str) -> Result<(), SPVError> {
```

Comment:
This whole function needs changing, you don't validate verification_params here anymore.

Reply:
done